### PR TITLE
Fix sidebar visibility issues for disability claims

### PR DESCRIPTION
**Pull request by bshyong**
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3076

### CODE REVIEW

**Review by melwoodard — COMMENTED** [review]
@bshyong this looks good. Is there an easy way to bring back the slide-out transition that used to happen when the menu is opened?

**Review by U-DON — APPROVED** [review]
LGTM. Might be a way to maintain the animated sliding effect, but not sure.

**Comment by bshyong** [comment]
I'll add this in—we can't hide the element completely if we want to keep transitions; we have to make it invisible with a height/width of 0 instead.

**Review by melwoodard — CHANGES_REQUESTED** [review]
The scale effect still isn't really right. If it isn't easy to do the slide-in without visible scaling, I say let's just keep it simple and take out the transition effect.

**Comment by bshyong** [comment]
@melwoodard updated—take a look at this and see what you think

http://recordit.co/d1dZ8pGOS2

File: src/sass/modules/_m-nav-sidebar.scss [hunk 1]
```diff
@@ -49,7 +39,19 @@
     }
 
     &--opened {
-      transform: translateX(0);
+      transform: scale(1);
+
+      &+ div:before {
```
Comment (U-DON) [inline src/sass/modules/_m-nav-sidebar.scss]: Super nitpick: usually I leave a space between the `&` and the operator.

**Review by melwoodard — COMMENTED** [review]
@bshyong better, but still seems funky with the scale effect stretching the menu out when it appears. I'm looking at this locally too and now the gray transparent overlay is on top of the menu, preventing me from interacting with it. I don't want to keep you spinning on this forever, so let's go back to the static transition for now and I can put in an issue to come back to the fine-tuning later.

**Comment by bshyong** [comment]
Updated—no animation now and we can improve this later and add animations back in

**Review by melwoodard — APPROVED** [review]
Thank you! Looks great, and I'll add the animation issue for the future.

**Pull request merged**